Formula/oh-my-posh.rb

```diff
@@ -21,7 +22,9 @@ class OhMyPosh < Formula
   def install
     ldflags = %W[
       -s -w
-      -X main.Version=#{version}
+      -X github.com/jandedobbeleer/oh-my-posh/src/build.Version=#{version}
+      -X github.com/jandedobbeleer/oh-my-posh/src/build.Date=#{time.iso8601}
+      -extldflags -static
```

Comment:
```suggestion
```
`-static` does not work on macOS (which is why the build failed).

Reply:
Thanks for the hint. Will remove it.